app/controllers/idv/enter_password_controller.rb

```diff
@@ -127,6 +127,7 @@ def confirm_current_password
     end
 
     def init_profile
+      reproof = current_user.has_proofed_before?
```

Comment:
I was going to ask if this could be moved into the conditional, but that wouldn't work because if we ran this check after creating a profile, it would always be true?

Reply:
yeah, exactly. i thought about checking to see if there was more than one profile that had the `activated_at` attribute, but i don't think what was happening was as clear